CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   ([#5560](https://github.com/facebook/jest/pull/5560))
 * `[jest-config]` Make it possible to merge `transform` option with preset
   ([#5505](https://github.com/facebook/jest/pull/5505))
+* `[babel-jest]` Remove `retainLines` argument to babel.
```

Comment:
"...from babel", or just "Remove usage of `retainLines`"

Reply:
My thinking was that people would go "what's `retainLines`, I've never passed that to jest before"